libs/portals/shared-modules/delegations/src/components/access/AccessCard.tsx

```diff
@@ -18,12 +18,10 @@ import { useLocale } from '@island.is/localization'
 import { AuthCustomDelegation } from '@island.is/api/schema'
 import { AuthDelegationType } from '@island.is/service-portal/graphql'
 import { useMemo } from 'react'
-import {
-  ServicePortalPath,
-  m as coreMessages,
-} from '@island.is/service-portal/core'
+import { m as coreMessages } from '@island.is/service-portal/core'
```

Comment:
Shouldn't we update all dependencies so that the Delegations module doesn't depend on any `service-portal` libraries?

Reply:
Discussed offline: Lets do this in another PR so we don't stop the show